std/process.d

```diff
@@ -1594,7 +1595,20 @@ version (Posix) @system unittest
     scope(exit) std.file.rmdirRecurse(directory);
     auto path = buildPath(directory, "tmp");
     std.file.write(path, null);
+    errno = 0;
     auto fd = open(path.tempCString, O_RDONLY);
+    if (fd == -1)
+    {
+        import core.stdc.string : strerror;
+        import std.stdio : stderr;
+        import std.string : fromStringz;
+
+        // For the CI logs
+        stderr.writefln("%s: could not open '%s': %s",
+            __FUNCTION__, path, strerror(errno).fromStringz);
```

Comment:
Hm, I was hoping `%s` would naturally print the zero-terminated string if passed a `const(char)*`. Sadly, it printe the address...

Reply:
The type `const(char)*` doesn't imply there's actually a zero-terminator, it would be unsafe to assume it is zero-terminated.